org.eclipse.m2e.core.ui/src/org/eclipse/m2e/core/ui/internal/components/PomHierarchyComposite.java

```diff
@@ -118,7 +118,7 @@ public String getText(Object element) {
       StringBuilder buffer = new StringBuilder();
       Model model = project.getProject().getModel();
       buffer.append(model.getGroupId()).append(" : ") //$NON-NLS-1$
-          .append(model.getArtifactId()).append(" : ") //$NON-NLS-2$
+          .append(model.getArtifactId()).append(" : ")
```

Comment:
The correction here is incorrect, it should be //$NON-NLS-1$
```suggestion
          .append(model.getArtifactId()).append(" : ") //$NON-NLS-1$
```